tests/core/storage/persistentstorage.js

```diff
@@ -6,11 +6,19 @@ describe('adding and removing data', () => {
   let updateCb;
 
   beforeEach(() => {
+    delete global.window.location;
```

Comment:
This is necessary to make changes to window.location.search within tests, as they are not natively changeable